source/blog/2016-05-30-dropping-dropbox.html.markdown

```diff
@@ -0,0 +1,99 @@
+
+1. The server syncs your files _in plaintext_, so an attacker who gains access to the server, or an NSA employee issuing a subpoena to Amazon for your instance's filesystem [?], now has all your files. 
+
+These problems can be pretty easily mitigated. First, though, I need to explain a bit more about how BitTorrent Sync works.
+
+## BitTorrent Sync
+
+The first time you start their app, you create an _identity_. Under the hood, BitTorrent Sync creates an [X.509 certificate](http://help.getsync.com/hc/en-us/articles/205451005) that's used to identify you and authenticate with computers you want to sync with, like your home desktop, work laptop, and smartphone. Nice. To link a new device to your identity, you use [a key in text or QR code format](http://help.getsync.com/hc/en-us/articles/205457815-Sync-Private-Identity-Linking-My-Devices). 
+
+Your identity syncs folders across devices. Unlike Dropbox where you have one big folder, BitTorrent Sync lets you have many folders with different permission/sharing settings. Any machine with your identity has access to all your files, unencrypted.
+
+The blog post I linked to suggests adding your identity to your new EC2 instance. I'm not sure this is wise, because an attacker who gains access to the instance now has your files. The process of adding the instance to your identity over unencrypted HTTP traffic also opens up concerns (I think the X.509 certificates mitigate this concern though [?]). My solution to this problem is to use _encrypted folders_.
+
+## Encrypted folders
+
+An [encrypted folder](http://help.getsync.com/hc/en-us/articles/207370466) is the perfect solution to this problem, and in fact, the docs recommend using encrypted folders for syncing with virtual private servers (which are [similar to EC2 instances](https://www.quora.com/Is-there-any-benefit-of-EC2-over-VPS-offered-by-dedicated-server-providers)). So let's do it!
+
+An encrypted folder is just like a regular folder that BitTorrent Sync synchronizes across devices, except that it adds encryption to the file contents it syncs. But encrypted folders separates the encryption keys for syncing the files and the keys for decrypting the files once they're sync'd. Your _identity_ automatically has access to the decryption key. 
+
+So my solution was **not adding my identity to the EC2 instance**. Instead, I gave it the "Encrypted Key" to sync with (under the share menu for the folder). This lets the EC2 instance securely sync the files _without_ having access to the decrypted version of the files. Nice. 
+
```

Comment:
Ah I see, and new folders added to one device won't be automatically sync'd with other clients, right?